packages/react-components/token-analyzer-preview/library/src/astAnalyzer.ts

```diff
@@ -102,15 +109,15 @@ function analyzeMergeClasses(sourceFile: SourceFile): StyleMapping[] {
 
       node.getArguments().forEach(arg => {
         // Handle direct style references
-        if (Node.isPropertyAccessExpression(arg) && arg.getText().startsWith('styles.')) {
-          mapping.baseStyles.push(arg.getName());
+        if (Node.isPropertyAccessExpression(arg)) {
```

Comment:
Nice fix, glad this was easy to unwind. Beyond `styles` how many other patterns do we see in code? Obviously, it's not enforced so being resilient here is crucial.